Docs: Update canvas docs for 10.2 (#76786)
grafana/grafana: docs/sources/panels-visualizations/visualizations/canvas/index.md

{{< video-embed src="/static/img/docs/canvas-panel/canvas-text-9-2-0.mp4" max-width="750px" caption="Text element demo" >}}

### Ellipse

> The ellipse element enables you to add a basic ellipse to the canvas. Ellipse elements support displaying text (both fixed and field data) as well as can change background color based on data thresholds.

Reviewer: Do you mean the background color of the ellipse element? I've edited assuming that meaning.
```suggestion
The ellipse element lets you add a basic ellipse to the canvas. An ellipse element can display text (both fixed and field data) and its background color can be changed based on data thresholds.
```

{{< figure src="/media/docs/grafana/canvas-server-element-9-4-0.png" max-width="750px" caption="Canvas server element" >}}

### Button

> The button element enables you to add a basic button to the canvas. Button elements support triggering basic, unauthenticated API calls. API settings are found in the button element editor. You can pass template variables in the API editor.

Reviewer: ```suggestion
The button element lets you add a basic button to the canvas. Button elements support triggering basic, unauthenticated API calls. API settings are found in the button element editor. You can also pass template variables in the API editor.
```

{{% admonition type="note" %}}

> To trigger an API call when clicking on a button in the canvas, [inline editing](#inline-editing) must be disabled.

Reviewer: ```suggestion
For a button click in a canvas to trigger an API call, [inline editing](#inline-editing) must be disabled.
```

Author: hm the wording here feels a little off - I will try to simplify this a bit
